build.gradle

```diff
@@ -68,6 +68,8 @@ dependencies {
         exclude group: 'com.fasterxml.jackson.core'
     }
     api group: 'org.json', name: 'json', version: '20250517'
+
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-ion:2.17.2'
```

Comment:
This is provided by the platform.

Reply:
It should be a dependency of the core so no need to add it.
If added, at least, it should not contain any version as the version is managed by the platform.